XMLs/ERJ-170-Model.xml

```diff
@@ -1074,7 +1074,7 @@
 
     <animation>
         <type>translate</type>
-        <object-name>reverser</object-name>
```

Comment:
maybe a good idea to rename the `OBJECT`s in the AC to `reverserL` and `reverserR` or sth like that, to make it clearer?